Sigma_specification.md

```diff
@@ -264,34 +244,38 @@ Creator of the rule. (can be a name, nickname, twitter handle...etc)
 
 **Attribute**: reference
 
-References to the source that the rule was derived from. These could be blog articles, technical papers, presentations or even tweets.
+References to the source that the rule was derived from. \
+These could be blog articles, technical papers, presentations or even tweets.
 
 ## Date (optional)
 
 **Attribute**: date
 
-Creation date of the rule. Use the format YYYY/MM/DD or YYYY-MM-DD
+Creation date of the rule. \
+Use the format YYYY/MM/DD or YYYY-MM-DD
```

Comment:
I would prefer to drop the slashed date format completely due to the ambiguity created for people using `YYYY/DD/MM`. `YYYY-MM-DD` is further the ISO standard.

Reply:
so "Use the ISO 8601 date with separator format YYYY-MM-DD"
And a small update in the V2_breaking_changes.md